tests/lib/devicetree/api/app.overlay

```diff
@@ -102,6 +102,7 @@
 			reg = <0x0 0x1000>;
 			interrupt-controller;
 			#interrupt-cells = <2>;
+			label = "Test No+Status1";
```

Comment:
how about adding in `!@#$%^&*()` into the string.

Reply:
Done